hikari/impl/bot.py

```diff
@@ -397,14 +404,19 @@ async def handle(name: str, awaitable: typing.Awaitable[typing.Any]) -> None:
             await coro
 
         # Join any shards until they finish
-        await aio.all_of(*(s.join() for s in self._shards.values()), timeout=len(self._shards))
+        try:
+            await aio.all_of(*(s.join() for s in self._shards.values()), timeout=len(self._shards))
+        except asyncio.TimeoutError:
+            _LOGGER.warning("shards didn't close down in time, force closing them...")
```

Comment:
what does force closing them mean here?

Reply:
Clearing the cache map and not giving them any more time to shutdown

Reply:
except that doesn't mean they'd have shutdown(?)